examples/dt/neutron/3cont_2comp_3net-ipv4-geneve-composable_provider_network

```diff
@@ -0,0 +1,66 @@
+# Deployed Topology 3cont_2comp_3net-ipv4-geneve-composable-vlan-provider-network
+
+## General information
+
+| Revision | Change              |  Date      |
+| -------: | :------------------ | :-----:    |
+|     v0.1 | Initial publication | 2024-01-16 |
+
+## Node topology
+
+| Node role                                  | bm/vm | amount |
+| ------------------------------------------ | ----- | ------ |
+| Openshift master/worker combo-node cluster | vm    | 3      |
+| Compute nodes                              | vm    | 2      |
+| Networker                                  | vm    | 3      |
+| CFME tester                                | vm    | 1      |
+
+
+## Services, enabled features and configurations
+
+| Service   | configuration                | Lock-in coverage?  |
+| --------- | ---------------------------- | ------------------ |
+| Neutron   | ML2/OVN, VLAN                | Must have          |
+|           | performance                  |                    |
+| Glance    | default                      | Must have          |
+| Cinder    | cinder_backup                | Must have          |
+| Keystone  | default                      | Must have          |
+| TLS       | default                      | Must have          |
+| Compute   | DifferentHostFilter          | Must have          |
+|           | neutron_azs_ovn              |                    |
+
+
+### Additional information 
+composable_roles
+
+#### Nova
+
+#### Cinder
+
+#### OVN
+add configuration of ipv4, OVNEncapTos=inherit
+
+## Considerations/Constraints
+
+
+## Testing tree
+Tempest and Rally
+
+| Test framework   | Stage to run | Special configuration | Test case to report |
+| ---------------- | ------------ | --------------------- | :-----------------: |
+| Tempest/neutron  | stage7       | Use rhel image        |      11223344       |
```

Comment:
A note for reviewers from other DFGs, all neutron d/s jobs use cirros as main(default) tempest image, rhel image is configured as an advanced image that is used only for tests where it is required. Tests that use  cirros and rhel are mixed in the same results file.
The above is related to all neutron jobs except the one that runs Rally, that one uses rhel as main tempest image. This was made because rally deploys some specific cirros image and there was some conflict during tempest deploy.

Reply:
This is good to read because it means we can combine these DTs with others. About rally, wasn't it possible to patch it to fix the conflict?